main.py

```diff
@@ -74,7 +67,8 @@ def parse_urls():
         availabilities = data["availabilities"]
         total = data["total"]
 
-        if not total and not availabilities:
+        # doctolib might return availabilities without any slots, just check for total slot number
+        if not total: # and not availabilities:
```

Comment:
sometimes requests like [this](https://www.doctolib.de/availabilities.json?start_date=2021-11-25&visit_motive_ids=3091828&agenda_ids=457424-457489-397846-404659-457448-457453-457410-457416-457439-457400-457429-457504-457497-457405-457404-457406-457409-457407-397844-457455-457511-397845-457436-457442-457487-457463-457477-457483-457470-457457-457493-457408&insurance_sector=public&practice_ids=158434&limit=4) lead to responses like
```json
{
  "availabilities": [
    {
      "date": "2021-11-25",
      "slots": []
    },
    {
      "date": "2021-11-26",
      "slots": []
    },
    {
      "date": "2021-11-27",
      "slots": []
    },
    {
      "date": "2021-11-28",
      "slots": []
    }
  ],
  "total": 0,
  "reason": "no_availabilities",
  "message": "Diese Termine stehen zu einem späteren Zeitpunkt wieder für eine Online-Buchung zur Verfügung."
}
```
But I've also seen the case that there was a `"total"` of 1 but no actual slot. Overall the API response seems to be quite... noisy.

For my own purpose I just want to be notified whenever they release a new batch of appointments so I even use. Otherwise there are just to many spam messages :D
```python
        try:
            total_i = int(total)
        except ValueError:
            println(f"Could not parse {total} as integer")
            continue

        # doctolib might return availabilities without any slots, just check for total slot number
        if total_i < 10:
            reason = data.get("reason")
            message = data.get("message")
            print(f"Total: {total_i}. Msg: {message}")
            continue
```